foam_controlwrapper/tests/test_foam_files.py

```diff
@@ -90,9 +90,9 @@ def test_modify_files(self):
         mesh_file = H5CUDS.open(os.path.join('foam_controlwrapper',
                                              'tests',
                                              'simplemesh.cuds'))
-        mesh_from_file = mesh_file.get_mesh(name)
+        mesh_from_file = mesh_file.get_dataset(name)
```

Comment:
Could you remove use of `simplmesh.cuds` (and the actual file) by using the approach you used in `FoamControlWrapperRunTestCase(unittest.TestCase)`.  So something like the following would replace lines 90-95:
        corner_points = [(0.0, 0.0, 0.0), (5.0, 0.0, 0.0),
                         (5.0, 5.0, 0.0), (0.0, 5.0, 0.0),
                         (0.0, 0.0, 1.0), (5.0, 0.0, 1.0),
                         (5.0, 5.0, 1.0), (0.0, 5.0, 1.0)]
        create_quad_mesh(name, self.wrapper, corner_points, 5, 5, 5)
        self.mesh_inside_wrapper = self.wrapper.get_dataset(name)

I guess this relates to comment https://github.com/simphony/simphony-openfoam/pull/53#issuecomment-151490598 in the sense that the H5File isn't really needed (only a openfoam-file/data-directory)